### PR TITLE
OSDOCS#9456: Adds ODC release notes

### PR DESCRIPTION
**Pull request by opayne1**
Adds ODC release notes

Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-9456

Link to docs preview:
https://71590--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-developer-perspective

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -82,6 +82,26 @@ You can now install a cluster on {ibm-cloud-name} in an environment with limited
 [id="ocp-4-15-web-console"]
 === Web console
 
+[id="ocp-4-14-developer-perspective"]
```
Comment (jerolimov) [inline release_notes/ocp-4-15-release-notes.adoc:85]: Super small thing:

```suggestion
[id="ocp-4-15-developer-perspective"]
```

**Comment by jerolimov** [comment]
From an eng. perspective this lgtm :+1:

**Review by christoph-jerolimov — CHANGES_REQUESTED** [review]
From an eng. perspective this lgtm 👍

There is just a tiny internal id wrong, see above.

**Comment by opayne1** [comment]
@sanketpathak could you help with a QE review of this PR?

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -82,6 +82,33 @@ You can now install a cluster on {ibm-cloud-name} in an environment with limited
...
+[id="ocp-4-15-software-supply-chain-enhancements"]
+===== Software Supply Chain Enhancements
+
+The *PipelineRun Details* page in the *Developer* or *Administrator* perspective of the web console provides an enhanced visual representation of PipelineRuns within a Project.
+
+For more information, see link:https://docs.openshift.com/pipelines/latest/about/understanding-openshift-pipelines.html[{pipelines-title}].
+
+[id="ocp-4-15-red-hat-developer-hub"]
+===== {rh-dev-hub} in the web console
+With this update, a quick start is now available for you to learn more about how to install and use the developer hub.
```
Comment (mramendi) [inline release_notes/ocp-4-15-release-notes.adoc:101]: I would make it "about installing and using". I think "about how to" is somewhat unwieldy.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -82,6 +82,33 @@ You can now install a cluster on {ibm-cloud-name} in an environment with limited
...
+
+For more information on the {rh-dev-hub}, see link:https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.0[Product Documentation for {rh-dev-hub}].
+
+[id="ocp-4-15-builds-suppot"]
+===== builds for {product-title} is supported in the web console
+
+With this update, builds for {product-title} 1.0 is supported in the web console. Builds is an extensible build framework based on the link:https://shipwright.io/[Shipwright project], which you can use to build container images on an OpenShift Container Platform cluster.
```
Comment (mramendi) [inline release_notes/ocp-4-15-release-notes.adoc:108]: I think that the "which" can be read a sreferring to the Shipwright project. I would split the sentence. Also, I would use {product-title}.

Comment (mramendi) [inline release_notes/ocp-4-15-release-notes.adoc:108]:
```suggestion
With this update, builds for {product-title} 1.0 is supported in the web console. Builds is an extensible build framework based on the link:https://shipwright.io/[Shipwright project]. You can use builds for {product-title} to build container images on an {product-title} cluster.
```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 4]
```diff
@@ -82,6 +82,33 @@ You can now install a cluster on {ibm-cloud-name} in an environment with limited
...
+
+For more information, see link:https://docs.openshift.com/builds/1.0/about/ob-release-notes.html[builds for {product-title}].
```
Comment (mramendi) [inline release_notes/ocp-4-15-release-notes.adoc:110]: For consistency with the Pipelines part, I would point to the overview and not to release notes: https://docs.openshift.com/builds/1.0/about/overview-openshift-builds.html

File: release_notes/ocp-4-15-release-notes.adoc [hunk 5]
```diff
@@ -82,6 +82,33 @@ You can now install a cluster on {ibm-cloud-name} in an environment with limited
...
+
+The *PipelineRun Details* page in the *Developer* or *Administrator* perspective of the web console provides an enhanced visual representation of PipelineRuns within a Project.
+
+For more information, see link:https://docs.openshift.com/pipelines/latest/about/understanding-openshift-pipelines.html[{pipelines-title}].
+
+[id="ocp-4-15-red-hat-developer-hub"]
+===== {rh-dev-hub} in the web console
+With this update, a quick start is now available for you to learn more about how to install and use the developer hub.
+
+For more information on the {rh-dev-hub}, see link:https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.0[Product Documentation for {rh-dev-hub}].
```
Comment (mramendi) [inline release_notes/ocp-4-15-release-notes.adoc:103]: I would use "about" not "on"

File: release_notes/ocp-4-15-release-notes.adoc (hunk 4 above)
Comment (mramendi) [inline release_notes/ocp-4-15-release-notes.adoc:110]:
```suggestion
For more information about builds for {product-title}, see link:https://docs.openshift.com/builds/1.0/about/overview-openshift-builds.html[builds for {product-title}].
```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 6]
```diff
@@ -82,6 +82,33 @@ You can now install a cluster on {ibm-cloud-name} in an environment with limited
...
+With this release, there are several updates to the *Developer* perspective of the web console.
+
+* Pipeline history and logs based on the data from Tekton Results are available in the dashboard without requiring PipelineRun CRs on the cluster.
+
+[id="ocp-4-15-software-supply-chain-enhancements"]
+===== Software Supply Chain Enhancements
+
+The *PipelineRun Details* page in the *Developer* or *Administrator* perspective of the web console provides an enhanced visual representation of PipelineRuns within a Project.
+
+For more information, see link:https://docs.openshift.com/pipelines/latest/about/understanding-openshift-pipelines.html[{pipelines-title}].
```
Comment (mramendi) [inline release_notes/ocp-4-15-release-notes.adoc:97]:
```suggestion
For more information about {pipelines-title}, see link:https://docs.openshift.com/pipelines/latest/about/understanding-openshift-pipelines.html[{pipelines-title}].
```

**Review by mramendi — COMMENTED** [review]
Please also note that the preview link does not seem to work for me. https://71590--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-web-console works.

As I am on peer review induction, @JoeAldinger will finish the review,

**Comment by sanketpathak** [comment]
LGTM 
QE approved!

File: release_notes/ocp-4-15-release-notes.adoc [hunk 7]
```diff
@@ -82,6 +82,33 @@ You can now install a cluster on {ibm-cloud-name} in an environment with limited
...
+
+[id="ocp-4-15-red-hat-developer-hub"]
+===== {rh-dev-hub} in the web console
+With this update, a quick start is now available for you to learn more about how to install and use the developer hub.
+
+For more information on the {rh-dev-hub}, see link:https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.0[Product Documentation for {rh-dev-hub}].
+
+[id="ocp-4-15-builds-suppot"]
```
Comment (JoeAldinger) [inline release_notes/ocp-4-15-release-notes.adoc:105]:
```suggestion
[id="ocp-4-15-builds-support"]
```

**Review by JoeAldinger — COMMENTED** [review]
A few suggestions, good PR.

**Pull request merged**